src/Modules/OrchardCore.Commerce.Payment.Stripe/Handlers/SubscriptionStripeWebhookEventHandler.cs

```diff
@@ -44,15 +44,14 @@ public async Task ReceivedStripeEventAsync(Event stripeEvent)
                 var user = await _cachingUserManager.GetUserByEmailAsync(invoice.CustomerEmail);
                 if (user == null)
                 {
-                    _logger.LogError(
+                    _logger.LogWarning(
```

Comment:
You know what? It might be okay to log it as error. Sorry....
```suggestion
                    _logger.LogError(
```

But the return delete is okay.